dashboard/test/ui/features/teacher_dashboard.feature

```diff
@@ -12,12 +12,6 @@ Feature: Using the teacher dashboard
     Then I click selector "div.title:contains('Student Accounts and Progress')"
     Then I wait until I am on "http://studio.code.org/home"
 
-  Scenario: Loading the teacher dashboard in the EU
-    Given I am on "http://code.org/"
-    And I am a teacher
-    And I am on "http://code.org/teacher-dashboard?no_home_redirect=1&force_in_eu=1"
-    Then I wait to see ".ui-test-gdpr-dialog"
-
```

Comment:
moved and expanded to ensure it is not shown if accepted, see line 31 of `gdpr_dialog.feature`